process/process_darwin.go

```diff
@@ -134,6 +134,8 @@ func (p *Process) createTimeWithContext(ctx context.Context) (int64, error) {
 	r, err := callPsWithContext(ctx, "etime", p.Pid, false)
 	if err != nil {
 		return 0, err
+	} else if len(r) == 0 || len(r[0]) == 0 {
```

Comment:
Useless `else` as the `if` just above exits the function with a `return`.

Reply:
Right but this puts the error validation checks in the same if-else block.  I think it makes it easier to read then two if blocks right after another but I can change it if you want.

Reply:
I would prefer if you changed it, thanks :)

Reply:
Okay this should work